readme.md

```diff
@@ -23,6 +23,18 @@ Requires macOS 10.13 or later.
 [![](https://linkmaker.itunes.apple.com/assets/shared/badges/en-us/macappstore-lrg.svg)](https://itunes.apple.com/no/app/gifski/id1351639930?mt=12)
 
 
+## System service
+
+Gifski provides a [system service](https://www.computerworld.com/article/2476298/os-x-a-quick-guide-to-services-on-your-mac.html) that lets you quickly convert a video to GIF from anywhere.
+
+You first have to enable it:
```

Comment:
Are you sure the user has to enable it manually? From experience, most services are auto-enabled when I install new apps.

This seems to confirm it: https://stackoverflow.com/questions/3859747/how-do-i-automatically-activate-an-item-in-the-os-x-services-menu

Reply:
Not totally sure, I had it unchecked 100% though. We may indicate in docs that we should make sure it's enabled in the settings.

Reply:
I have manually verified that services are enabled by default.